src/shared/components/mutationTable/column/FunctionalImpactColumnFormatter.tsx

```diff
@@ -59,315 +56,232 @@ class FunctionalImpactColumnTooltip extends React.Component<
         };
     }
 
+    renderHeaderIcon(title: string, imageSrc: string, onMouseOver: () => void) {
+        return (
+            <th>
+                <span
+                    style={{ display: 'inline-block', width: 22 }}
+                    title={title}
+                    onMouseOver={onMouseOver}
+                >
+                    <img height={14} width={14} src={imageSrc} alt={title} />
+                </span>
+            </th>
+        );
+    }
+
+    renderImpactRow(
+        iconClass: string,
+        impactData: string[],
+        showMutationAssessor: boolean
+    ) {
+        return (
+            <tr>
+                <td>
+                    <span
+                        className={classNames(
+                            annotationStyles['annotation-item-text'],
+                            iconClass
+                        )}
+                    >
+                        <i className="fa fa-circle" aria-hidden="true"></i>
+                    </span>
+                </td>
+                {showMutationAssessor && (
+                    <td className={classNames(iconClass)}>{impactData[0]}</td>
+                )}
+                <td className={classNames(iconClass)}>{impactData[1]}</td>
+                <td className={classNames(iconClass)}>{impactData[2]}</td>
+                <td className={classNames(iconClass)}>{impactData[3]}</td>
+            </tr>
+        );
+    }
+
     legend() {
         const showMutationAssessor = shouldShowMutationAssessor();
+        // Each line in the legend table uses the same style (mutationAssessorStyles)
```

Comment:
maybe rename `mutationAssessorStyles` to `functionalImpactStyles` or something like that?